src/main/webapp/style.css

```diff
@@ -0,0 +1,75 @@
+.container {
+    font-family: "Montserrat";
+    display: flex;
+    flex-direction: column;
+}
+
+.home-title {
+    color: rgb(16, 89, 16);
+    font-size: 5em;
+    margin: 2em 0 0;
+    text-align: center;
+}
+
+.home-subtitle {
+    color: rgb(19, 73, 19);
+    font-size: 1em;
+    text-align: center;
+}
+
+.enter-btn-div {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+/* Buttons */
+.btn {
+    border: none;
+    border-radius: 20px;
+    font-family: "Montserrat";
+    height: 5vh;
+    margin: 0.5em 0;
+    width: 20vw;
+}
+
+.login-btn {
+
```

Comment:
Since the rule is empty try to include it in the next PR and hold off on committing it until it's got something in it.